lib/Minilla/Project.pm

```diff
@@ -709,6 +709,9 @@ sub regenerate_readme_md {
                     $image_uri->host('kritika.io');
                     $image_uri->path("/users/$user_name/repos/$user_name+$repository_name/heads/master/status.svg");
                     push @badges, "[![Kritika Status]($image_uri)]($build_uri)";
+                } elsif ($service_name eq 'github-actions') {
+                    my $workflow_name = 'test'; ## XXX: fixed
```

Comment:
I don't think fixed workflow names are very useful.
How do we customize them?

Reply:
Yes! I think so too.

I have some ideas:

1. Includes the workflow name to `$service_name` (e.g. `github-actions/test`, `github-actions:test` or `github-actions(workflow=test)`)
2. Add on new `badge_options` setting in `minil.toml`, and add the workflow name to here (e.g. `badge_options = { github-actions = { workflow = 'test' } }`)
3. Supports `test` workflow name only. All users should use `test` as testing workflow name. (maybe it's incorrect, but we can think about it later)

IMO...

The plan `1` is easy, but it have poor extensibility, and may become a technical debt.
The plan `2` is correct probably, but it may break the consistency of the `minil.toml` configuration style, and it may become a technical debt.
The plan `3` is poor, but we can postpone the decision. We should discuss about it when needed. (YAGNI)

Which is better do you think? or do you have any other ideas?

Reply:
I prefer 1 `github-actions/test`.

Reply:
OK. Thank you!